### DISCUSSION TITLE
Drag and drop modules

### DISCUSSION THREAD

**Discussion by MattatDirectus** [directus/directus#16850] [Feature Requests]
Small life improvement - if we could drag and drop re-order the modules on the left instead of navigating over to the settings page to re-order.

**Comment by rijkvanzanten** [comment]
Would you expect the order to persist for just your user, or for everybody? Right now the order in settings controls it for everybody, but it might be a bit confusing if the inline drag and drop changes the settings for everybody 🤔

**Reply by MattatDirectus** [reply]
User-level would be preferable vs. project-level. 

Kind of like Discord/Slack's server reorganization. 

Would be cool to "clean up" my personal workspace and personalize my own Directus instance (I also like the idea of folders for the modules that I don't use much). 

<img width="63" alt="Screenshot 2022-12-23 at 3 50 34 PM" src="https://user-images.githubusercontent.com/118303535/209404419-a4eaa7b7-cd03-4589-96f3-c0db70b25f14.png">

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
